stations.csv

```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74451;Schlatt Paradies (Fähre);schlatt-paradies-fahre;;;47.68527;8.674222;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8530837;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74451;
 74452;Schlatt TG Bahnhof;schlatt-tg-bahnhof;;;47.679333;8.687562;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8573491;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74452;
 74453;Havneby;havneby;;;55.086166;8.569992;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8600921;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74453;
+74454;Garraf;garraf;7171703;;41.254712;1.904007;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74454;
+74455;Els Monjos;els-monjos;7172203;;41.316708;1.662007;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74455;
+74456;Lavern Subirats (Apd);lavern-subirats-apd;7172206;;41.395755;1.768162;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74456;
```

Comment:
Lavern Subirats (Apd) - is this the station name?

Reply:
This should be fixed